config/config.go

```diff
@@ -657,8 +659,9 @@ var defaultConf = Config{
 		Engines: []string{"tikv", "tiflash", "tidb"},
 	},
 	Experimental: Experimental{
-		AllowsExpressionIndex: false,
-		EnableGlobalKill:      false,
+		AllowsExpressionIndex:  false,
+		EnableGlobalKill:       false,
+		EnableEnhancedSecurity: false,
```

Comment:
I don't think we need to mark it as 'Experimental' since the goal is GA for the next release(the risk should be low IMO).